olaaf_django/sync_hashes.py

```diff
@@ -87,22 +96,51 @@ def _insert_diff_hashes(repo, prev_commit, current_commit):
       # if file was added or modified, calculate the new hash
       file_type = 'html' if file_name.endswith('.html') else 'pdf'
       if file_type == 'html':
-        url = path.rsplit('.', 1)[0]
```

Comment:
this will almost certainly be wrong for edge cases. Instead just grab url from https://github.com/cityofsanmateo/law-html/blob/f9c78256eeec88e8ed431e35a16d3abeac1c942a/us/ca/cities/san-mateo/code/1.01.050.html#L19

(you can use https://docs.python.org/3/library/urllib.parse.html to easily get just the path)